### PR TITLE
drivechain-server: fix enforcer proto path, startup instructions

### PR DESCRIPTION
**Pull request by arnemolland**
The enforcer proto file path changed in [bip300301_enforcer](https://github.com/LayerTwo-Labs/bip300301_enforcer/commit/5b7c7c8452252ffd464cef16a76c11fa12374753) and have been updated.

Also updated the binary path for the enforcer as the downloaded binary name differed slightly from the instructions.

### CODE REVIEW

**Comment by arnemolland** [comment]
btw there's some other stuff that broke with the path changing, don't merge this yet

File: drivechain-server/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ $ drivechaind \
 
 #
 # Then, start the bip300301_enforcer
-$ ./enforcer/bin/enforcer \
```
Comment (octobocto) [inline drivechain-server/README.md:32]: Ja, dette var feil! My bad

**Comment by arnemolland** [comment]
Should be good now

**Comment by octobocto** [comment]
perfect. Do you know if it's possible to exclude gen/ from dart fix? It's kindof annoying to have it always fail on regeneration...

**Comment by arnemolland** [comment]
> perfect. Do you know if it's possible to exclude gen/ from dart fix? It's kindof annoying to have it always fail on regeneration...

Should be possible, let me check and I'll add it here

> perfect. Do you know if it's possible to exclude gen/ from dart fix? It's kindof annoying to have it always fail on regeneration...

Yes, we did already ignore most protobuf stubs but were missing `.pbgrpc.dart` files so I added that to all clients

**Comment by octobocto** [comment]
perfect!

**Pull request merged**